tests/unit/test_core/test_auth.py

```diff
@@ -141,18 +132,17 @@ def test_02_list_users_request(self, mock_jwt_secret):
         """Test auth list users endpoint."""
         invalid_header = {"Authorization": "Bearer invalidtoken"}
         schema = {"users": list}
-        password = "password".encode()
-        # Patch _list_users_callback event callback.
-        event_boxes = [self.user_data,
-                       {"username": "authtempuser2",
-                        "email": "tempuser2@kytos.io",
-                        "password": hashlib.sha512(password).hexdigest()}]
-        self.patched_events.append({'_list_users_callback': event_boxes})
+        event_boxes = {'users': [
```

Comment:
Let's call it response instead of `event_boxes` anything related to box or storehouse let's completely replace with the current structures that you're migrating.

Reply:
Changed.